src/components/global/Playground/index.tsx

```diff
@@ -591,27 +591,29 @@ export default function Playground({
   }
 
   useEffect(() => {
-    const codeSnippets = {};
+    const codeSnips = {};
```

Comment:
Changing the variable to make it easier to read the difference between this and the `codeSnippets` state.